### PR TITLE
🆙 Have version bump script also update bug template

### PR DESCRIPTION
**Pull request by henryboldi**
We're forgetting about updating our bug template, so let's make it automatic!

### CODE REVIEW

**Review by greg-nagy — CHANGES_REQUESTED** [review]
For me the yq command does not add the new version but duplicates the block and adds the version to the 2nd block. 

Also the `preversion` script creates a new file name 1 with the path for `yq`.
Not sure why, but the script is working though, just creates a new file. 

Now that you are in this PR anyway, can you please add brave browser to the list? :) 🙏

CleanShot 2022-01-04 at 15 06 39

File: scripts/update-version.sh [hunk 1]
```diff
@@ -18,3 +18,7 @@ fi
 # Update manifest version and stage for version bump commit.
 echo "$(jq ".version=\"$npm_package_version\"" manifest/manifest.json | yarn -s prettier --stdin-filepath=manifest.json)" > manifest/manifest.json
 git add manifest/manifest.json
+
+# Add version to bug template and stage for version bump commit.
+yq e -i '.body[] |= select(has("id")) |= select(.id == "version").attributes.option = ["Community Edition v'$npm_package_version'"] + select(.id == "version").attributes.options' .github/ISSUE_TEMPLATE/BUG.yml
```
Comment (Shadowfiend) [inline scripts/update-version.sh:23]:
```suggestion
yq e -i '.body[] |= select(has("id")) |= select(.id == "version").attributes.options = ["Community Edition v'$npm_package_version'"] + select(.id == "version").attributes.options' .github/ISSUE_TEMPLATE/BUG.yml
```

Comment (henryboldi) [reply scripts/update-version.sh]: Oh my lord. Thank you sir

**Comment by Shadowfiend** [comment]
Letting @greg-nagy wrap this one up.

**Review by greg-nagy — APPROVED** [review]
it wooooorkkss ..... yeeeey :D

**Pull request merged**